content/en/blog/2023/exponential-histograms.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Exponential Histograms
+date: 2023-05-22
+author: '[Daniel Dyla](https://github.com/dyladan)'
+spelling: cSpell:ignore Dyla quantiles subsetting Ruslan Vovalov Ganesh Vernekar
+canonical_url: https://dyladan.me/histograms/2023/05/04/exponential-histograms/
+---
+
+Previously, in [Why Histograms?][] and [Histograms vs Summaries][], I have gone
+over the basics of histograms and summaries and the tradeoffs, benefits, and
+limitations of each of them. Because they're easy to understand and demonstrate,
```

Comment:
```suggestion
Previously, in [Why Histograms?][] and [Histograms vs Summaries][], I went
over the basics of histograms and summaries, explaining the tradeoffs, benefits, and
limitations of each. Because they're easy to understand and demonstrate,
```